docs/mkdocs.yml

```diff
@@ -40,6 +40,7 @@ nav:
   - Getting Started:
       - Quickstart using Kind: Getting-Started/user-quickstart-kind.md 
       - Quickstart using OpenShift: Getting-Started/user-quickstart-openshift.md 
+      - Quickstart for testing: Getting-Started/user-quickstart-test.md 
```

Comment:
do we need to expose this?

Reply:
I was wondering about that. Makes it easy for me to find and test. Maybe put in a contributor section?

Reply:
for now its fine - lets leave it, but we could move it later.  Not the most important detail.